### PR TITLE
Add-linter-gitignore-and-template

### PR DESCRIPTION
**Pull request by Gdiazdiaz**
### Tasks accomplished

- [x] Add a linter file for HTML&CSS
- [x] Add a .gitignore file with node_modules inside
- [x] Add an HTML file with a header, headline, and links to the style sheet, media, and google fonts
- [x] Add a CSS file with the parameters from the Figma template

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+  width: 66px;
+  height: 24px;
+  left: 625px;
+  top: 38px;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  text-align: right;
```
Comment (Owusu-Desmond) [inline style.css]: - [x] Please form the project requirement you should stick to the [figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=70%3A1).
  - You must stick to the design as much as possible
    | Your Design (No hamburger and design do not match) | Figma Design(With hamburger) |
    | :---:   | :---: | 
    | Screen | Screenshot fro |

File: README.md [hunk 2]
```diff
@@ -1,75 +1,35 @@
...
 
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
-
-## 🤝 Contributing
-
-Contributions, issues, and feature requests are welcome!
-
-Feel free to check the [issues page](../../issues/).
```
Comment (Owusu-Desmond) [inline README.md]: - [x] Please form the project requirement you should follow the [README template](https://github.com/microverseinc/readme-template) and add the link to your project issue page under the the `## 🤝 Contributing` section of your readme file

File: README.md [hunk 3]
```diff
@@ -1,75 +1,35 @@
...
-👤 **Author1**
-
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
+👤 **Author**
 
-👤 **Author2**
+- GitHub: [@Gdiazdiaz](https://github.com/Gdiazdiaz)
+- LinkedIn: [Glenda Diaz](www.linkedin.com/in/glendadiazz)
```
Comment (Owusu-Desmond) [inline README.md]: - [x] Please kindly replace `Author` with your name

**Review by Owusu-Desmond — CHANGES_REQUESTED** [review]
Hi @Gdiazdiaz :wave:  

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
:heavy_check_mark:  Good PR description
:heavy_check_mark:  No linter errors

## Required Changes ♻️
_Check the comments under the review._

- [x] From the project requirement, you should setup up both [html&css](https://github.com/microverseinc/linters-config/blob/master/html-css) linters but you only setup CSS linter. By setting up the HTML linter you should have [.hintrc](https://github.com/microverseinc/linters-config/blob/master/html-css/.hintrc) in the root directory of your project.

- [x] Please Note that a Good commit message should be capitalized and summarized.  But from your commit message history you have some capitalized and some not capitalized.


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 2 above)
Comment (Gdiazdiaz) [reply README.md]: The link to my project? to the main branch?

Comment (Gdiazdiaz) [reply README.md]: Can you explain how I didn't follow the template?

Comment (Owusu-Desmond) [reply README.md]: - [x] Please make sure when the `issues page` link is clicked it takes you to the issues page of your project. You can do this by replacing `../../issues/` with your project issues page link.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+            <img id="header-img2" src="media/image_geometry_2.png">
+        </div>
+        <nav>
+            <a>
+                <img id="menu" src="media/ic_menu.png">
+            </a>
+        </nav>
+    </header>
+    <headline class="headline">
+        Glenda Diaz
```
Comment (leonardodiasb) [inline index.html:22]: - [OPTIONAL] This is the main title of the page and it should be wrapped in an `h1` tag. If you want that google (and other search engines) knows about a specific topic, in this case, you, use an h1 tag. [Check this link](https://www.reliablesoft.net/h1-tag/) for more info.

**Review by leonardodiasb — APPROVED** [review]
# Status: ✔️ Project Approved ✔️ 

Hello @Gdiazdiaz  🔥 

You worked hard in all the instructions that the previous Code Reviewers give you👍🏽 

Now you can merge this branch 🚀 

## Highlights

✔️ Good PR description
✔️ No linter errors

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**